sites/partners/src/listings/PaperListingDetails/sections/DetailApplicationDates.tsx

```diff
@@ -32,6 +32,11 @@ const DetailApplicationDates = () => {
     () =>
       listing.events
         .filter((item) => item.type === ListingEventType.openHouse)
+        .sort((a, b) => {
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore
+          return new Date(a.startTime) - new Date(b.startTime)
+        })
```

Comment:
I think you don't need to ignore TS warning here, a `getTime()` method might help to get numeric value, like:

```
return new Date(a.startTime).getTime() - new Date(b.startTime).getTime()
```

so now that's the arithmetic operation.